packages/react-router-dom/modules/NavLink.js

```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { Route, matchPath } from "react-router";
+import {
+  __RouterContext as RouterContext,
+  Route,
```

Comment:
`<Route />` can be removed as it is obsolete.

Reply:
It's in the PropTypes.

Reply:
Oops

Reply:
@timdorr Should it really be there? In both cases it's just PropTypes.bool. And it's rather unlikely to change.